src/Donations/resources/components/DonationsListTable.tsx

```diff
@@ -143,6 +144,14 @@ export default function DonationsListTable() {
             filterSettings={filters}
             paymentMode={!!window.GiveDonations.paymentMode}
         >
+            {!!window.GiveDonations.manualDonations && (
+                <a
+                    className={tableStyles.addFormButton}
+                    href={window.GiveDonations.adminUrl + 'edit.php?post_type=give_forms&page=give-manual-donation'}
```

Comment:
I think we can use [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals).

```suggestion
                    href={`${window.GiveDonations.adminUrl}edit.php?post_type=give_forms&page=give-manual-donation`}
```

Reply:
Resolved: https://github.com/impress-org/givewp/pull/6638/commits/ce95534952ac1b9d67838acdb83ba241c5eca253